app/src/main/java/io/github/muntashirakon/AppManager/details/AppDetailsActivity.java

```diff
@@ -157,7 +159,7 @@ protected void onCreate(Bundle savedInstanceState) {
                     }
                 }
                 // Check for the existence of package
-                viewPager.setAdapter(new AppDetailsFragmentPagerAdapter(fragmentManager));
+                viewPager.setAdapter(new AppDetailsFragmentPagerAdapter(this));
```

Comment:
Attach `TabLayoutMediator` after this line instead of at line 110.

Reply:
I forgot this as well 🤐